lib/graphql/define/defined_object_proxy.rb

```diff
@@ -11,6 +11,15 @@ def types
         GraphQL::Define::TypeDefiner.instance
       end
 
+      def use(plugin_class, *args, **kwargs)
```

Comment:
Is there a use case in mind for `*args`? If not, I think we should drop it until we see a clear need. I can't _think_ of anything where `**kwargs` wouldn't be sufficient for now.